### ISSUE TITLE
variadic argument got an error

### ISSUE THREAD

**Issue by lydiandy** [vlang/v#5001]
**V version:** 
latest: V 0.1.27 c95a113
**OS:** 
macos
**What did you do?**
here is the example code:
```
module main

fn main() {
	add('a')
	add('a','b','c')
}

fn add(column string, other_columns ...string) {
	println(column)
	println(other_columns)
}
```

**What did you expect to see?**
can println 
 
**What did you see instead?**
error: too many arguments to function call, expected single argument 's', have 2 arguments
                strings__Builder_write(&sb, string_str(it.args[i], 0));
                                            ~~~~~~~~~~             ^

**Comment by Gladear** [comment]
Duplicate of #4999

**Comment by spytheman** [comment]
This is a different, although related issue, caused by buggy auto generation of str methods for variadic arguments.

**Comment by spytheman** [comment]
Fixed in ddcb5f7 , See https://github.com/vlang/v/blob/master/vlib/v/tests/vargs_auto_str_method_and_println_test.v .

**Comment by lydiandy** [comment]
@spytheman thanks
